src/main/resources/locale_pt_PT.properties

```
@@ -0,0 +1,196 @@
+command.setup.dialog.rolesFormat=Por favor defina-os como <$words.role$> <$words.reputation$>
+command.setup.dialog.selectLanguage=Please select your language.
+command.setup.dialog.setupComplete=Setup complete!
+command.setup.dialog.starting=Arrancar setup.
+command.setup.dialog.welcome=Bem-vindos à configuração do bot de reputação.
+command.thankwords.description=Configurar palavras.de.agradecimento
+command.thankwords.error.patternNotFound=Pattern não existe.
+command.thankwords.sub.add=Add another pattern. Padrões são interpretados como Regex. Um padrão não pode conter espaços.
+command.thankwords.sub.add.added=Adicionar padrão %REGEX%.
+command.thankwords.sub.check=Cheque se uma mensagem daria reputação.
+command.thankwords.sub.check.confidence=Confidence Score: %SCORE%
+command.thankwords.sub.check.match.answer=Aresposta Match
+command.thankwords.sub.check.match.fuzzy=Fuzzy Match
+command.thankwords.sub.check.match.mention=Mention Match
+command.thankwords.sub.check.match.noMatch=No Match
+command.thankwords.sub.check.notConfident=Confidence Score is to low. Não será dada uma reputação.
+command.thankwords.sub.check.reference=Reference Message: [Saltar](%URL%)
+command.thankwords.sub.check.result=Donatador: %DONATOR%%NReceptor: %RECEIVER%
+command.thankwords.sub.list=Listar todos os padrões
+command.thankwords.sub.list.list=Following Pattern are active:
+command.thankwords.sub.loadDefault=Load default thankswords for a language.
+command.thankwords.sub.loadDefault.added=Added the following thankwords:
+command.thankwords.sub.loadDefault.available=Línguas disponíveis:
+command.thankwords.sub.remove=Delete a pattern
+command.thankwords.sub.remove.removed=Removed pattern %PATTERN%.
+conversation.cancel=cancelar
+conversation.canceled=Conversation canceled.
+conversation.exit=saída
+conversation.finished=Conversation finished.
+conversation.inProgress=Já está em curso uma conversação neste canal. Pode sair desta conversa digitando $conversation.exit$ ou $conversation.cancel$.
+conversation.start=Iniciando uma nova conversação.\n Você pode deixar a conversa a qualquer momento digitando $conversation.exit$ ou $conversation.cancel$.
+error.commandNotFound=Command não encontrado.
+error.invalidArguments= Argumentos inválidos
+error.invalidChannel=Invalid Channel
+error.invalidMessage=Mensagem inválida
+error.invalidNumber=Número inválido
+error.invalidRegex=Invalid regex pattern.
+error.invalidRole=Invalid Role
+error.missingPermission=Sinto falta da permissão %PERM% para funcionar correctamente.
+error.missingPermissionChannel= Isto deve-se à definição de permissões de anulação no canal %CHANNEL%.
+error.missingPermissionGuild=É devido a definições de permissão globais.
+error.noValidChannel=Não são fornecidos canais válidos.
+error.notABoolean=%INPUT% is not a boolean (true|false).
```

Looks untranslated

DeepL suggestion:
```suggestion
error.notABoolean=%INPUT% não é um booleano (true|false).
```